### ISSUE TITLE
Error: `env: node\r: No such file or directory`

### ISSUE THREAD

**Issue by cedx** [cedx/reverse-proxy.js#1]
Any `reverse-proxy` command I try always results in the same error:

```
$ reverse-proxy --help
env: node\r: No such file or directory
```

**Comment by cedx** [comment]
The problem is be related to the packaging phase: I'm developing on Windows. Effectively I should have done my npm publish command on a Unix system.
I will fix this in the next release.

**Comment by cedx** [comment]
Fixed in version 0.3.0.
